src/rgw/rgw_sal.cc

```diff
@@ -209,6 +209,10 @@ rgw::sal::Store* StoreManager::init_storage_provider(const DoutPrefixProvider* d
                           << " user in sfs error r=" << r << dendl;
       }
     }
+    if (store->initialize(cct, dpp) < 0) {
```

Comment:
now `initialize` doesn't do much, but I guess in the future it could. 
anyway, you're right... if we need it in the future we can add it.

changed.